src/app.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+/*
+The file is concerned about maintaining the angular
+part of the app
+*/
+let invApp = angular.module("invertedIndex", []);
+
+
+invApp.controller("invertedController", ["$scope", function($scope) {
+  let invIndex = new Index();
+  $scope.uploadedFiles = {};
+  $scope.allFlag = false;
+  $scope.allFilesIndexed = {};
+  /**
+  $scope.getIndex is executed when the getIndex button is clicked
+  */
+  $scope.getIndex = function() {
+    let fileChoice = $scope.uploadSelected;
+    if(fileChoice === undefined) {
```

Comment:
**Code Quality**  undefined is a falsy value, it always returns false. So if **!fileChoice** might be ideal here, only if your variable returns other falsy values and you are checking undefined explicitly.